symphony/bdk/core/service/message/message_service.py

```diff
@@ -466,7 +466,7 @@ async def search_messages_one_page(skip, limit):
 
     @retry
     async def update_message(self, stream_id: str, message_id: str, message: Union[str, Message], data=None,
-                             version: str = "") -> V4Message:
+                             silent=True, version: str = "") -> V4Message:
```

Comment:
I'm wondering if adding this parameter doesn't break the backward compatibility. I would rather add a new method that takes this new argument, and I call it in this one with the default silent value (True).